deploy/initialize.ts

```diff
@@ -56,13 +56,14 @@ const createDAO = async (deployments: DeploymentsExtension, deployer: string, si
     })
     .find(dao => dao !== undefined);
 
-  const aclAddress = await kernel.acl();
+  const dao = Kernel__factory.connect(daoAddress, signer);
```

Comment:
Good catch! for the record it can be found [here](https://github.com/AragonBlack/fundraising/blob/73ca077608c9db353240487b2d0486166255681b/apps/aragon-fundraising/test/helpers/setup.js#L85)